components/Modals/AddGameMasterModal.tsx

```diff
@@ -92,20 +91,11 @@ export const AddGameMasterModal: React.FC = () => {
       {...{
         isOpen: addGameMasterModal?.isOpen,
         onClose: addGameMasterModal?.onClose,
-        header: 'Make Address Eligible as GameMaster',
+        header: 'GameMaster Added',
```

Comment:
Would just change this to "Add GameMaster," since the header renders before any GameMaster is actually added

Reply:
Tweaked in 815918e916da0a901497f7102dfae42841ea444c